archinstall/__init__.py

```diff
@@ -41,7 +45,15 @@ def initialize_arguments():
 	args = parser.parse_args()
 	if args.config is not None:
 		try:
-			config = json.load(args.config)
+			# First, let's check if this is a URL scheme instead of a filename
+			parsed_url = urllib.parse.urlparse(args.config)
+
+			if not parsed_url.scheme:  # The Profile was not a direct match on a remote URL, it must be a local file.
+				config = json.load(args.config)
```

Comment:
I think we might need to create the file object here if we're not using FileType, as FileType automatically parses the argument as a file and creates a file object for us

Reply:
Yes, I think it's correct now.